protostar/testing/fuzzing/strategies/integers.py

```diff
@@ -1,18 +1,23 @@
 from typing import Optional
 
-from hypothesis.strategies import SearchStrategy, integers
+from hypothesis.strategies import SearchStrategy, integers as h_integers
 from starkware.cairo.lang.compiler.ast.cairo_types import CairoType, TypeFelt
 
 from protostar.testing.fuzzing.exceptions import SearchStrategyBuildError
 from protostar.testing.fuzzing.strategy_descriptor import StrategyDescriptor
 
 
+def integers(
+    min_value: Optional[int] = None,
+    max_value: Optional[int] = None,
+) -> StrategyDescriptor:
+    return IntegersStrategyDescriptor(min_value, max_value)
+
+
 class IntegersStrategyDescriptor(StrategyDescriptor):
     # NOTE: Keeping these names matching arguments of Hypothesis' ``integers`` strategy,
     #   so that Hypothesis' exceptions (which contain these names) make sense for our users.
-    def __init__(
-        self, min_value: Optional[int] = None, max_value: Optional[int] = None
-    ):
+    def __init__(self, min_value: Optional[int], max_value: Optional[int]):
```

Comment:
A user will need to pass `None` explicitly. Is it intended?

Reply:
Init is not user facing function now, there is a wrapper named `felts` (the same as strategy, it's important) that has defaults and proper interface